Switch default Parquet writer version to PARQUET_1_0

majetideepak opened the pull request:
We must switch the default to PARQUET_1_0 since its the most compatible/stable Parquet version.
Parquet V2 spec is not official. The various implementations out there have potential compatibility issues.
Trino and Spark default to PARQUET_1_0.

Resolves https://github.com/prestodb/presto/issues/23366

## Description
<!---Describe your changes in detail-->

## Motivation and Context
<!---Why is this change required? What problem does it solve?-->
<!---If it fixes an open issue, please link to the issue here.-->

## Impact
<!---Describe any public API or user-facing feature change or any performance impact-->

## Test Plan
<!---Please fill in how you tested your change-->

## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== RELEASE NOTES ==

General Changes
* Switch default Parquet writer version to PARQUET_1_0. 
Please set the hive config `hive.parquet.writer.version = PARQUET_2_0` for old behavior. :pr:`23369`

```

tdcmeehan commented:
Can you add a message in the release note on how to change it back to 2.0 if the user so desires?

steveburnett commented:
> Can you add a message in the release note on how to change it back to 2.0 if the user so desires?

Maybe this could be documented in https://prestodb.io/docs/current/presto_cpp/properties.html ?

majetideepak commented:
@tdcmeehan I updated the comment as well as the release note on how to get the old behavior.
@steveburnett This is not specific to Prestissimo. I updated the default values in the connector documentation.

steveburnett reviewed (dismissed):
LGTM! (docs) 
Pull branch, local doc build, looks good. Thanks!

majetideepak commented:
@tdcmeehan, @steveburnett  Can you approve this one last time? All tests are now passing. Thank you!

steveburnett reviewed (approved):
LGTM! (docs)

Pull updated branch, new local docs build, re-review by request, no problems, looks good.
